packages/blockchain/xrp/src/lib/xrp.sdk.ts

```diff
@@ -0,0 +1,29 @@
+import {BlockchainXrpService} from '@tatumio/api-client'
+import {SDKArguments} from '@tatumio/abstract-sdk'
+import {abstractBlockchainSdk} from "@tatumio/shared-blockchain-abstract";
+import {Blockchain} from "@tatumio/shared-core";
+
+const blockchain = Blockchain.XRP
+
+export const TatumXrpSDK = (args: SDKArguments) => {
+  const api = BlockchainXrpService
+
+  return {
+    ...abstractBlockchainSdk({...args, blockchain }),
+    api,
```

Comment:
What about bootstrap KMS and transactions?

Reply:
Yes, I noticed in the morning, I am working on that.